doc/build-instructions/Build_Instructions_V11.md

```diff
@@ -808,7 +808,7 @@ The following sections cover the whole installation & building steps for all req
 :rocket:
 For the cross-compilation, the first thing is to install all required software packages on your system before building the JDK. When it comes to Fedora/RISC-V, you need to boot the OS image via `QEMU` to login the target system for installation.
 
-[1] First of all, download & compile the `QEMU` source on your host system as follows:
+[1] First of all, download & compile the latest version of `QEMU` source on your host system as follows:
 ```
 wget https://download.qemu.org/qemu-5.0.0-rc0.tar.xz
 tar xvJf qemu-5.0.0-rc0.tar.xz
```

Comment:
I see `5.0.0-rc4`, dated April 22, and `5.0.0` April 28: should this be updated to use version 5.0.0?

Reply:
Updated to `5.0.0` given that the link at https://www.qemu.org/download/#source was updated to download the latest version (5.0.0).